versions/draft-2/parsers/antlr4/WdlLexer.g4

```diff
@@ -0,0 +1,194 @@
+lexer grammar WdlLexer;
```

Comment:
For consistency with the above changes you should change this to `WdlDraft2Lexer`